src/server/zset_family_test.cc

```diff
@@ -578,4 +578,54 @@ TEST_F(ZSetFamilyTest, BlockingTimeout) {
   EXPECT_THAT(resp0, ArgType(RespExpr::NIL_ARRAY));
 }
 
+TEST_F(ZSetFamilyTest, ZDiffError) {
+  RespExpr resp;
+
+  resp = Run({"zdiff", "-1", "z1"});
+  EXPECT_THAT(resp, ErrArg("value is not an integer or out of range"));
+
+  resp = Run({"zdiff", "0"});
+  EXPECT_THAT(resp, ErrArg("wrong number of arguments"));
+
+  resp = Run({"zdiff", "0", "z1"});
+  EXPECT_THAT(resp, ErrArg("value is not an integer or out of range"));
```

Comment:
Some of the error messages are not identical with `redis`. I played around with different commands like `ZUNION` etc. Should the error messages be 1-1 with redis for compatibility?

Reply:
Generally no